ramani-maplibre/src/main/java/org/ramani/compose/MapLibre.kt

```diff
@@ -145,6 +145,7 @@ fun MapLibre(
         currentSources,
         currentLayers,
         currentImages,
+        currentStyleBuilder,
```

Comment:
This makes us recompose the whole thing when the style changes. Maybe there is a more elegant way to do that, but it will do for now. Anyway the style is not meant to change often.

Reply:
@JonasVautherin Yes, I can confirm that once the map shows, the symbol appears, and then about a second later, disappears and reappears. I guess that is when we recompose after the style has loaded. I think this is totally fine, as it coincides with other tiles on the map appearing.